suite-common/message-system/config/config.v1.json

```diff
@@ -147,6 +147,64 @@
                 }
             }
         },
+        {
+            "conditions": [
+                {
+                    "settings": [
+                        {
+                            "matic": true
+                        }
+                    ],
+                    "environment": {
+                        "desktop": "<=24.9.2",
```

Comment:
i think this can be `*` as web, it should be filtered out by `settings` `matic: true` which does not exist in new versions